configure

```diff
@@ -1064,6 +1064,13 @@ fi
 if sh ./hasgot2 -i math.h $mathlib expm1 log1p hypot copysign fma; then
   inf "expm1(), log1p(), hypot(), copysign(), fma() found."
   echo "#define HAS_C99_FLOAT_OPS" >> s.h
+  if test "$system" != "cygwin" || test "$arch" != "amd64" ; then
```

Comment:
Why using FMA emulation for amd64 arch?

Reply:
Cygwin amd64 is the architecture where the underlying `fma` function isn't working.

Reply:
OK, I see now, thanks.